cmd/addblock/import.go

```diff
@@ -297,9 +298,45 @@ func (bi *blockImporter) Import() chan *importResults {
 // newBlockImporter returns a new importer for the provided file reader seeker
 // and database.
 func newBlockImporter(db database.DB, r io.ReadSeeker) (*blockImporter, error) {
+	// Create the transaction and address indexes if needed.
```

Comment:
How about the following comment change so it remains correct if new ones are added and fixes the fact it is now incorrect?

```Go
// Create the various indexes as needed.
```